src/04_printing.py

```diff
@@ -12,6 +12,12 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print('x is %s, y is %s, z is "%s"' % (x, round(y, 2), z))
+
 # Use the 'format' string method to print the same thing
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is ' + str(x) + ', y is ' + str(round(y, 2)) + ', z is "' + z + '"')
+
+# Finally, print the same thing using an f-string
+
+print(f'x is {x}, y is {round(y, 2)}, z is "{z}"')
```

Comment:
Good work! We can also use floating point specification like so: `print("x is {}, y is {:.2f}, z is \"{}\"".format(x, y, z))`